SPECS/malaga/malaga.spec

```
@@ -1,13 +1,12 @@
-Vendor:         Microsoft Corporation
-Distribution:   Mariner
-Name:           malaga
-Version:        7.12 
-Release:        31%{?dist}
 Summary:        A programming language for automatic language analysis
-
+Name:           malaga
+Version:        7.12
+Release:        32%{?dist}
 License:        GPLv2+
-URL:            http://home.arcor.de/bjoern-beutel/malaga/
-Source0:        http://home.arcor.de/bjoern-beutel/malaga/%{name}-%{version}.tgz
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://dynalabs.de/mxp/malaga/
+Source0:        https://code.launchpad.net/ubuntu/+archive/primary/+sourcefiles/malaga/7.12-7.1/malaga_7.12.orig.tar.gz
```

Please use macros at least for the version, so updates are easier.

```suggestion
Source0:        https://code.launchpad.net/ubuntu/+archive/primary/+sourcefiles/malaga/%{version}-7.1/malaga_%{version}.orig.tar.gz
```

Also, is there anything wrong with this:
```suggestion
Source0:        https://dynalabs.de/mxp/assets/malaga-%{version}.tgz
```

The signature seems to match and it might be easier to update in the future, since it doesn't have the `-7.1` path in the URL.

---

Earlier, this URL was resolving as invalid for me, but now seems to work. I'll test it locally again and see what happens when I push.